.dockerignore

```diff
@@ -0,0 +1,6 @@
+.dockerignore
+**.pyc
+**/__pycache__
+.gitignore
+.git
+./allennlp/data/fields/__pycache__
```

Comment:
ah oops!  I misunderstood @jkinkead 's comment.  This is just a debugging line I was using earlier.  Your are both absolutely right--it should not be here.